tests/h/accounts/views_test.py

```diff
@@ -241,24 +79,26 @@ def test_post_no_event_when_validation_fails(self,
         assert not loginevent.called
         assert not notify.called
 
-    def test_post_redirects_when_validation_succeeds(self, pyramid_config, pyramid_request):
+    def test_post_redirects_when_validation_succeeds(self,
+                                                     form_validating_to,
+                                                     pyramid_config,
+                                                     pyramid_request):
         pyramid_config.testing_securitypolicy(None)  # Logged out
         controller = views.AuthController(pyramid_request)
-        controller.form = form_validating_to(
-            {"user": FakeUser(username='cara')})
+        controller.form = form_validating_to({"user": FakeUser(username='cara')})
```

Comment:
I'm just going to make the point again that including formatting changes in diffs like this is not helpful to reviewers. Please try and resist the temptation! 

Ideally a PR is either refactoring, in which case the goal of the reviewer is to identify changes that are not strictly behaviour-preserving, or a PR is changing behaviour, in which case the goal of the reviewer is to understand the changes that have been made.